diff --git a/dom/bindings/Bindings.conf b/dom/bindings/Bindings.conf
--- a/dom/bindings/Bindings.conf
+++ b/dom/bindings/Bindings.conf
@@ -318,16 +318,24 @@ DOMInterfaces = {
 'ConvolverNode': {
     'implicitJSContext': [ 'buffer' ],
 },
 
 'Coordinates': {
     'headerFile': 'nsGeoPosition.h'
 },
 
+'Credential' : {
+    'workers': False,
+},
+
+'CredentialContainer' : {
+    'workers': False,
+},
+
 'Crypto' : {
     'headerFile': 'Crypto.h'
 },
 
 'CSS': {
     'concrete': False,
 },
 
@@ -493,16 +501,20 @@ DOMInterfaces = {
     'nativeType': 'mozilla::dom::workers::ExtendableEvent',
 },
 
 'ExtendableMessageEvent': {
     'headerFile': 'mozilla/dom/ServiceWorkerEvents.h',
     'nativeType': 'mozilla::dom::workers::ExtendableMessageEvent',
 },
 
+'FederatedCredential' : {
+    'workers': False,
+},
+
 'FetchEvent': {
     'headerFile': 'ServiceWorkerEvents.h',
     'nativeType': 'mozilla::dom::workers::FetchEvent',
     'binaryNames': {
         'request': 'request_'
     },
     'implicitJSContext': [ 'respondWith' ],
 },
@@ -714,16 +726,21 @@ DOMInterfaces = {
     'concrete': False
 },
 
 'LegacyMozTCPSocket': {
     'headerFile': 'TCPSocket.h',
     'wrapperCache': False,
 },
 
+'LocallyStoredCredential' : {
+    'workers': False,
+    'concrete': False
+},
+
 'LocalMediaStream': {
     'headerFile': 'DOMMediaStream.h',
     'nativeType': 'mozilla::DOMLocalMediaStream'
 },
 
 'Location': {
     'nativeType': 'nsLocation',
 },
@@ -913,16 +930,20 @@ DOMInterfaces = {
 'OfflineResourceList': {
     'nativeType': 'nsDOMOfflineResourceList',
 },
 
 'PaintRequestList': {
     'headerFile': 'mozilla/dom/PaintRequest.h',
 },
 
+'PasswordCredential' : {
+    'workers': False
+},
+
 'Path2D': {
     'nativeType': 'mozilla::dom::CanvasPath',
     'headerFile': 'CanvasPath.h'
 },
 
 'PeerConnectionImpl': {
     'nativeType': 'mozilla::PeerConnectionImpl',
     'headerFile': 'PeerConnectionImpl.h',
