diff -r 8f21d63b310a dom/webidl/Navigator.webidl
--- a/dom/webidl/Navigator.webidl	Sat May 09 08:00:10 2015 -0700
+++ b/dom/webidl/Navigator.webidl	Mon May 11 17:07:14 2015 +0200
@@ -435,8 +435,18 @@
 #endif
 
 #ifdef NIGHTLY_BUILD
 partial interface Navigator {
   [Func="Navigator::IsE10sEnabled"]
   readonly attribute boolean mozE10sEnabled;
 };
 #endif
+
+// W3C Credential Management
+[NoInterfaceObject, Exposed=Window]
+interface NavigatorCredentialContainer {
+  [Throws]
+  readonly attribute CredentialContainer credentials;
+};
+Navigator implements NavigatorCredentialContainer;
+
+
